src/main/java/io/github/railroad/utility/Terminal.java

```diff
@@ -0,0 +1,37 @@
+package io.github.railroad.utility;
+
+import static com.sun.javafx.util.Utils.*;
+import static java.lang.Runtime.getRuntime;
+
+/**
+ * Opens the terminal.
+ *
+ * @author saksham4106, temedy
+ */
+public interface Terminal {
+    /**
+     * the command that opens the terminal in windows
+     */
+    String WINDOWS = "cmd.exe /c start",
+    /**
+     * the command that opens the terminal in linux
+     */
+    LINUX = "bash -c start %s %s",
```

Comment:
This seems a bad idea. if a user has fish, or zsh, or heck even just sh, this will fail hard.
Consider using the $TERM or $XTERM environment variables that are set by all available shells?

Reply:
I don't use linux, so I don't know how to make that work. I'll look into it.